api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -390,7 +390,8 @@ message FilterStateRule {
 
   // A map of string keys to requirements. The string key is the string value
   // in the FilterState with the name specified in the *name* field above.
-  map<string, JwtRequirement> requires = 3;
+  map<string, JwtRequirement>
```

Comment:
Not sure why this has changed, but it shouldn't.
Can you double check if it happens due to proto_format?

Reply:
Yeah. It is proto format thing only. I will revert. Sorry. I did not notice this.